nav/Nav.screen.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Menu from '../screen/Menu/Menu.screen';
+import AboutMe from '../screen/AboutMe/AboutMe.screen';
+import colors from '../res/colors';
+
+
+const Stack = createNativeStackNavigator();
+
+const Nav = () => {
+    return (
+        <NavigationContainer>
+            <Stack.Navigator>
+                <Stack.Screen options={{
+                    headerStyle: {
+                        backgroundColor: colors.purple,
+                    },
+                    headerTintColor: colors.white,
+                    headerTitleStyle: {
+                        fontWeight: 'bold',
+                    }
+                }} name="Menu" component={Menu} />
```

Comment:
create folder 'config', inside of it create file 'constants.js' ,then define MENU and PROFILE constants. Import here them and use as name={MENU}.
If you want to change title for Screens. Use property options. Find more information in api of react-navigation about  StackNavigator.